x-pack/plugins/infra/public/components/metrics_explorer/chart_context_menu.test.tsx

```diff
@@ -24,10 +24,31 @@ describe('MetricsExplorerChartContextMenu', () => {
       />
     );
 
+    component.find('button').simulate('click');
+    const menuItems = component.find('.euiContextMenuItem__text');
+    expect(menuItems.length).toBe(3);
+    expect(menuItems.at(0).text()).toBe('Add filter');
+    expect(menuItems.at(1).text()).toBe('Open in Visualize');
+    expect(menuItems.at(2).text()).toBe('View metrics for host');
```

Comment:
Do we want to test the correctness of the link href too to ensure the date is properly represented? Alternatively, a functional test could cover this too, but would be way more expensive.

Reply:
I created a proxy function which creates the link to make it easier to test.